extension/chrome/settings/setup/setup-recover-key.ts

```diff
@@ -60,6 +60,7 @@ export class SetupRecoverKeyModule {
       const options: SetupOptions = {
         submit_main: false, // todo - reevaluate submitting when recovering
         submit_all: false,
+        key_type: 'rsa4096', // TODO(@limonte): ask Tom is this right?
```

Comment:
Same here, importing key from a backup - not creating any.

Reply:
Done :+1: